### DISCUSSION TITLE
2 threads; Thread A is working, Thread B is stealing. B missed the 128 tasks added by A in an instant, B goes to sleep, and A needs to execute the 128 tasks independently?

### DISCUSSION THREAD

**Discussion by sxstd001** [Kotlin/kotlinx.coroutines#3915] [General]
2 threads; Thread A is working, Thread B is stealing. B missed the 128 tasks added by A in an instant, B goes to sleep, and A needs to execute the 128 tasks independently?

**Comment by qwwdfsad** [comment]
Could you please elaborate on your question and provide more details?
